packages/front-end/tsconfig.json

```diff
@@ -17,7 +17,9 @@
     "incremental": true,
     "baseUrl": "./",
     "paths": {
-      "@/*": ["./*"]
+      "@/*": ["./*"],
+      "@back-end/*": ["../back-end/*"],
+      "@shared/*": ["../shared/*"]
```

Comment:
I'm not sure if this is correct ... We want to import from `'shared/*'` and not `../shared/*` if I'm not mistaken. The former is package reference (see frontend package.json dependencies  for `shared`), which should reference the compiled TS in your local dev. The latter is relative, and I'm not sure if it works the same or not. (Correct me if I'm wrong please!)

Reply:
Work. I think you're right. Have removed the ts path config and substituted all `@shared` for `shared`.